### DISCUSSION TITLE
Displaying content from _index.md files

### DISCUSSION THREAD

**Discussion by mdecimus** [thuliteio/doks#880] [Q&A] [answered]
Hi,

First of all thank you for making this beautiful template!

I'd like to ask if there is a way to display content in a section's _index.md? Let's say I have the following structure using the ``doc`` type:

```
 /lorem/
 /lorem/ipsum
 /lorem/dolor
```

I need to be able to display custom content in ``lorem/_index.md`` instead of the default table with the subsections. Is there an easy way to do it or would that require modifying the templates? 
I tried renaming ``lorem/_index.md`` to ``lorem/index.md`` but the subsections are no longer displayed.

Thank you!

**Comment by h-enk** [comment, marked as answer]
Thanks! You can add content to e.g. `./content/en/info/prologue/_index.md`. That will then be displayed like so:

![Snag_1da1fd0f](https://user-images.githubusercontent.com/3902872/188385434-9131e8d1-91a1-470b-9d2b-cdd2c4978a35.png)

If you'd want more customizations, you'd have to update `./layouts/docs/list.html`.

**Comment by mdecimus** [comment]
I'll edit ``./layouts/docs/list.html`` as I need to remove the table with the subsections.

Thank you!
